components/brave_rewards/browser/rewards_service.h

```diff
@@ -149,8 +149,13 @@ class RewardsService : public KeyedService {
 
   // Returns the Rewards feature version associated with the current user. The
   // returned `base::Version` will always be valid.
+  // DEPRECATED: Use `GetUserType` instead.
   virtual base::Version GetUserVersion() const = 0;
```

Comment:
Do we want to get rid of this function?

Reply:
I thought that Android was using it but it appears not. I'll remove it.